src/cc/bcc_common.h

```diff
@@ -69,7 +70,8 @@ struct bpf_insn;
 int bcc_func_load(void *program, int prog_type, const char *name,
                   const struct bpf_insn *insns, int prog_len,
                   const char *license, unsigned kern_version,
-                  int log_level, char *log_buf, unsigned log_buf_size);
+                  int log_level, char *log_buf, unsigned log_buf_size,
+                  const char *dev_name);
```

Comment:
could you make dev_name with default value nullptr?

Reply:
I don't think I can for these functions because they have C linkage. I can---and just did---for method `BPFModule::bcc_func_load` however.

Reply:
Sorry. I actually indeed mean the change in bpf_module.h and it looks like you just did. Great!